src/plugins/displayLayout/DisplayLayoutToolbar.js

```diff
@@ -196,24 +196,62 @@ define([], function () {
                     options: [
                         {
                             value: false,
-                            icon: "icon-grid-snap-no",
-                            title: "Do not snap to grid"
+                            icon: "icon-grid-snap-to",
+                            title: "Grid snapping enabled"
                         },
                         {
                             value: true,
-                            icon: "icon-grid-snap-to",
-                            title: "Snap to grid"
+                            icon: "icon-grid-snap-no",
+                            title: "Grid snapping disabled"
                         }
                     ]
                 };
+               let x = {
+                        control: "input",
+                        type: "number",
+                        domainObject: selectedParent,
+                        property: function () {
+                            return getPath() + ".x";
+                        },
+                        label: "X:",
+                        title: "X position"
+                    },
+                    y = {
+                        control: "input",
+                        type: "number",
+                        domainObject: selectedParent,
+                        property: function () {
+                            return getPath() + ".y";
+                        },
+                        label: "Y:",
+                        title: "Y position",
+                    },
+                    width = {
+                        control: 'input',
+                        type: 'number',
+                        domainObject: selectedParent,
+                        property: function () {
+                            return getPath() + ".width";
+                        },
+                        label: 'W:',
+                        title: 'Resize object width'
+                    },
+                    height = {
+                        control: 'input',
+                        type: 'number',
+                        domainObject: selectedParent,
+                        property: function () {
+                            return getPath() + ".height";
+                        },
+                        label: 'H:',
+                        title: 'Resize object height'
+                    };
```

Comment:
Indentation seem to be off here?

Reply:
It was and I just pushed a fixed version.